### PR TITLE
Qt: Add Video Capture button to toolbar

### PR DESCRIPTION
**Pull request by kamfretoz**
### Description of Changes

This PR adds the Video Capture button into the toolbar for quicker access.
image

### Rationale behind Changes

Easier to access the video capture rather than going through Tools -> Video Capture

### Suggested Testing Steps

QoL 100

### CODE REVIEW

File: pcsx2/GS/GSCapture.cpp [hunk 1]
```diff
@@ -289,13 +289,15 @@ bool GSCapture::LoadFFmpeg(bool report_errors)
...
-						"  libswscale: {}\n"
-				"  libswresample: {}\n"), LIBAVCODEC_VERSION_MAJOR, LIBAVFORMAT_VERSION_MAJOR, LIBAVUTIL_VERSION_MAJOR,
+		Host::ReportErrorAsync(TRANSLATE_SV("GSCapture", "Failed to load FFmpeg"),
+			fmt::format(TRANSLATE_FS("GSCapture", "You may be missing one or more files, or are using the incorrect version. This build of PCSX2 requires:\n"
+				"  libavcodec: {}\n"
+				"  libavformat: {}\n"
+				"  libavutil: {}\n"
+				"  libswscale: {}\n"
+				"  libswresample: {}\n\n"
+				"Please refer to our official documentations for more information."),
```
Comment (Mrlinkwii) [inline pcsx2/GS/GSCapture.cpp:299]: "Please refer to our official documentation for more information"  , but otherwise look good

Comment (kamfretoz) [reply pcsx2/GS/GSCapture.cpp]: Done

**Review by TheTechnician27 — APPROVED** [review]

File: pcsx2-qt/MainWindow.cpp [hunk 2]
```diff
@@ -712,14 +712,14 @@ void MainWindow::updateAdvancedSettingsVisibility()
...
+void MainWindow::onVideoCaptureToggled(bool checked)
 {
 	if (!s_vm_valid)
 		return;
 
 	// Reset the checked state, we'll get updated by the GS thread.
-	QSignalBlocker sb(m_ui.actionToolsVideoCapture);
-	m_ui.actionToolsVideoCapture->setChecked(!checked);
+	QSignalBlocker sb(m_ui.actionVideoCapture);
+	m_ui.actionVideoCapture->setChecked(false);
```
Comment (F0bes) [inline pcsx2-qt/MainWindow.cpp:722]: If `onCaptureStopped` and `onCaptureStarted` slots set the checked state. Is it possible to remove these two lines here?

Comment (kamfretoz) [reply pcsx2-qt/MainWindow.cpp]: Done

**Review by F0bes — APPROVED** [review]
lgtm and works as expected :+1:

**Pull request merged**